changelog/unreleased/enhancement-table-sorting

```diff
@@ -0,0 +1,6 @@
+Enhancement: Table sorting
+
+The latest version of OcTable and OcTableFiles had no more sorting.
+With this pr we bring it back by simply adding a sortable true to the table fields.
```

Comment:
```suggestion
We brought it back. It can be enabled by simply adding a `sortable: true` to table fields.
```
Changelog text should be a self contained short summary of a PR, not mentioning `this pr`. Otherwise it suggests that you need to look at the PR to understand the changelog.